src/createServer.js

```diff
@@ -1,3 +1,50 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const errorMessages = require('./errorMessages');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [path, queryString] = req.url.split('?');
+    const text = path.slice(1);
+    const params = new URLSearchParams(queryString);
+    const targetCase = params.get('toCase');
+    const responseErrors = {
+      errors: [],
+    };
+
+    res.setHeader('Content-Type', 'application/json');
+
+    if (!text) {
+      responseErrors.errors.push({ message: errorMessages.missingText });
+    }
+
+    if (!targetCase) {
+      responseErrors.errors.push({ message: errorMessages.missingTargetCase });
+    }
```

Comment:
Move it to the separate function
```suggestion
    res.setHeader('Content-Type', 'application/json');
```